artifact.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+PUBKY="https://api.bintray.com/orgs/jfrog/keys/gpg/public.key"
+REPOURL="https://jfrog.bintray.com/artifactory-debs"
+
+ echo "Installing Java for Jfrog Artifactory"
+ apt-get update
+ apt-get install default-jdk -y
+ if [ $? -eq 0 ]; then
+	
+	wget -qO - $PUBKY |  apt-key add - 
+	add-apt-repository "deb [arch=amd64] $REPOURL $(lsb_release -cs) main"
+	apt-get update
+	
+	echo "Installing Jfrog artifactory"
+	apt install jfrog-artifactory-oss -y
+	
+	echo "Setting up services!!!"
+	for i in start enable status ; do
+		systemctl $i artifactory.service
```

Comment:
script halts at status ( status was active). Add a grep condition to check whether the status was active.